config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -148,29 +153,34 @@ periodics:
     preset-aws-credential: "true"
   branches:
     - release-0.4
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
```

Comment:
Should this be `release-0.2` here instead?

Reply:
Or is capi even needed here? it wasn't listed in the repos removed below

Reply:
sorry, deleted